components/image_resolution_extraction/fondant_component.yaml

```diff
@@ -0,0 +1,19 @@
+name: Image resolution extraction
+description: Component that extracts image resolution data from the images
+image: ghcr.io/ml6team/image_resolution_extraction:latest
+
+consumes:
+  images:
+    fields:
+      data:
+        type: binary
+
+produces:
+  images:
+    fields:
+      width:
+        type: int16
+      height:
+        type: int16
+      data:
+        type: binary
```

Comment:
For components that only add columns, is there a need to specify existing columns in the produces section?

cc @RobbeSneyders

Reply:
In theory not, but I'm not sure if it works in practice already if you leave them out.